File: C/1864.c
#include <stdio.h>
int main()
{
    int val, i, c, d;
    char frase[40] = "LIFE IS NOT A PROBLEM TO BE SOLVED";
    scanf("%d", &val);
    for(i = 0; i < val; i++) {
        printf("%c", frase[i]);
    }
    printf("\n");
    return 0;
}

File: Java/1864.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
		Scanner leitor = new Scanner(System.in);
		String frase = "LIFE IS NOT A PROBLEM TO BE SOLVED";
		String fr[] = new String[34];
		fr = frase.split("");
		int n = leitor.nextInt();
		for(int i = 0 ; i < n; i++) {
			System.out.printf("%s", fr[i]);
		}
		System.out.println("");
    }
}